front/src/applications/operationalStudies/utils.ts

```diff
@@ -290,3 +292,14 @@ export const isTooFast = (
   }
   return false;
 };
+
+export const formatSpaceTimeDataToProjectPathTrainResult = (
```

Comment:
You don't need this function, you can simply replace trainName by name when you construct your SpaceTimeData ;)
The SpaceTimeData is built only at one place, in SimulationResultsV2. Change the type SpaceTimeData and then change the function getSpaceTimeChartData

Reply:
Done thank you